testing/citest/tests/google_server_group_test.py

```diff
@@ -266,6 +275,13 @@ def clone_server_group(self):
      .list_resource(self.__mig_manager_name, **self.__mig_manager_kwargs)
      .contains_path_value('baseInstanceName', self.__cloned_server_group_name))
 
+    (builder.new_clause_builder('Instance Template Created', retryable_for_secs=150)
```

Comment:
Is this clause something like "Instance template preserved"?
If the value here is tied to above, then use a variable or constant to capture it rather than duplicating it.